src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -371,19 +363,12 @@ private void LogHostFunctionErrors()
         /// Gets metadata collection of functions and proxies configured.
         /// </summary>
         /// <returns>A metadata collection of functions and proxies configured.</returns>
-        private IEnumerable<FunctionMetadata> GetFunctionsMetadata(bool workerIndexing)
+        private IEnumerable<FunctionMetadata> GetFunctionsMetadata()
         {
             IEnumerable<FunctionMetadata> functionMetadata;
-            if (workerIndexing)
-            {
-                _logger.LogInformation("Worker indexing is enabled");
-                functionMetadata = _functionMetadataManager.GetFunctionMetadata(forceRefresh: false, dispatcher: _functionDispatcher);
-            }
-            else
-            {
-                functionMetadata = _functionMetadataManager.GetFunctionMetadata(false);
-                _workerRuntime = _workerRuntime ?? Utility.GetWorkerRuntime(functionMetadata);
-            }
+
+            functionMetadata = _functionMetadataManager.GetFunctionMetadata(forceRefresh: false);
```

Comment:
Should it be `True` as we discussed in design sync?

Reply:
We can leave this as false as scriptstartuptypelocator has set it to true. That piece of code runs everytime script host starts